prep-javascript-variables/index.html

```diff
@@ -0,0 +1,24 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>JavaScript Variables</title>
+</head>
+<body>
+  <script>
+    /* your code here */
+    var fullName = 'Cody Martin';
+    console.log('fullName', fullName);
+    var numberOfPets = 1;
+    console.log('numberOfPets:', numberOfPets);
+    var likesVegetables = 'true';
+    console.log('likesVegetables:', likesVegetables);
+    var nothing = 'null';
+    console.log('nothing:', nothing);
```

Comment:
Hi Cody, be sure to leave out the quotes from `true` and `null` so that the browser doesn't interpret them as strings.

Reply:
![Screen Shot 2022-04-25 at 10 09 10 PM](https://user-images.githubusercontent.com/96100245/165226326-b313f25e-4460-4b88-987d-17a9295934d9.png)